pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[tool.black]
+line-length = 120
+exclude = '''
+/(
+    \.git
+  | \.github
+  | \.idea
+  | \.vscode
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+  | services
+  | fetch
+  | \.temp
+)/
+'''
+
+[build-system]
+requires = ["setuptools>=75.7.0"]
```

Comment:
[python-setuptools on Arch only has 75.2.0 which does not work with this requirement](https://archlinux.org/packages/extra/any/python-setuptools/).